docs/internal.md

```diff
@@ -64,6 +67,46 @@ yarn package
 ```
 
 This will generate a file `dist/web-ext-artifacts/sporran-???.zip` ready to be uploaded to stores.
+This version will only connect to _KILT Spiritnet_ blockchain via predefined node endpoints.
+
+### For the internal version
+
+Run the following commands:
+
+```
+yarn install
+yarn build-internal
+yarn package
+```
+
+This will generate a file `dist/web-ext-artifacts/sporran-???.zip` ready for developers to use.
+This version will connect to any node endpoints the user inputs and by default to the _KILT Peregrine_ blockchain.
+
+## Uploading to the release's site
```

Comment:
```suggestion
## Uploading to the releases page
```
Unlike in German, in English "site" in relation to the internet pretty much always means everything on a given domain, not a single page

Reply:
ok. 

changed.